c2corg_ui/templates/404.html

```diff
@@ -0,0 +1,56 @@
+<%inherit file="base.html"/>
+<%namespace file="helpers/common.html" import="show_title"/>
+<%block name="pagetitle"><title ng-bind="mainCtrl.page_title('Page not found')">${show_title('Page not found')}</title></%block>
+
+
+## waypoint lost in the bermuda triangle!
+<%block name="moduleConstantsValues">
+module.value('mapFeatureCollection', {
+"type": "FeatureCollection",
+  "features": [
+      {
+        "geometry": {
+            "type": "Point",
+            "coordinates": [-7601449.065694831, 3050623.9688091446]
+        },
+        "type": "Feature",
+        "properties": {
+            "type": "virtual",
+            "title": "404",
+            "module": "waypoints"
+        }
+    }
+]
+});
+</%block>
+
+<header class="page-main-title">
+  <h3><span translate>unknown page</span></h3>
+</header>
+
+<section class="not-found-section">
+  <h1 class="green-text text-center text-uppercase">oops</h1>
+  <h4 translate class="text-center">The page you are looking for does not exist or is broken.</h4>
+  <h4 class="text-center"><span translate>You can go to the</span> <a href="javascript:history.go(-1)" translate>previous page</a> <span translate>or try to go to other pages:</h4>
+  <ul class="not-found-pages">
+    <a class="list-item text-center" href="/"><span translate>Home</span><br><span class="glyphicon glyphicon-home"></span></a>
```

Comment:
Shouldn't you rather use the usual Pyramid route names with ``${request.static_url(...``?